config/presets/representation-conversion.json

```diff
@@ -37,6 +37,25 @@
       ]
     },
 
+    {
+      "@id": "urn:solid-server:default:ContentTypeReplacer",
+      "@type": "ContentTypeReplacer",
+      "ContentTypeReplacer:_replacements": [
+        {
+          "ContentTypeReplacer:_replacements_key": "text/turtle",
+          "ContentTypeReplacer:_replacements_value": "application/trig"
+        },
+        {
+          "ContentTypeReplacer:_replacements_key": "application/n-triples",
+          "ContentTypeReplacer:_replacements_value": "application/n-quads"
+        },
```

Comment:
Actually, @rubensworks, I tried to make this (following https://componentsjs.readthedocs.io/en/latest/configuration/components/object_mapping/#example-single-array-constructor):
```suggestion
        {
          "ContentTypeReplacer:_replacements_key": "application/n-triples",
          "ContentTypeReplacer:_replacements_value": {
            "elements": [
              "text/turtle",
              "application/trig",
              "application/n-quads"
            ]
          }
        },
```

but then I get:

```
Error: could not instantiate server from /Users/ruben/Documents/UGent/Solid/solid-community-server/config/config-default.json
TypeError: value.elements.map is not a function
    at /Users/ruben/Documents/UGent/Solid/solid-community-server/node_modules/componentsjs/lib/factory/UnnamedComponentFactory.js:79:51
```

where `value` is:

```
Resource {
  value: '_:b8bnode202',
  termType: 'BlankNode',
  'https://linkedsoftwaredependencies.org/vocabularies/object-mapping#elements': [
    Resource {
      value: '_:b10bnode202',
      termType: 'BlankNode',
      'http://www.w3.org/1999/02/22-rdf-syntax-ns#rest': [Array],
      __listRest: [Array]
    }
  ],
  elements: Resource {
    value: '_:b10bnode202',
    termType: 'BlankNode',
    'http://www.w3.org/1999/02/22-rdf-syntax-ns#rest': [ [Resource] ],
    __listRest: [ [Resource] ]
  }
}
```

So somehow it was not seen as an array. Can I type it as such?
Note that the constructor in question is overloaded.

@joachimvh This can be added later though.

Reply:
Looks like a bug indeed, which may very well already work in the new version :-)

Reply:
Using the chaining there is now you could add `n-triples` -> `turtle` and the trig option would also be included so could solved by adding that one for now.

Reply:
Done!